modules/rosa-scaling-worker-nodes.adoc

```diff
@@ -0,0 +1,59 @@
+:_module-type: PROCEDURE
+//Specify the system-module-type as either "CONCEPT, PROCEDURE, or REFERENCE"
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-managing-worker-nodes.adoc
+
+[id="rosa-scaling-worker-nodes_{context}"]
+= Scaling worker nodes
+
+[role="_abstract"]
+Worker nodes can be scaled manually if you do not want to configure node autoscaling.
+
+.Procedure
+
+. To get a list of the machine pools in a cluster, enter the following command. Each cluster has a default machine pool that is created when you create a cluster.
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID        AUTOSCALING   REPLICAS    INSTANCE TYPE  LABELS    TAINTS   AVAILABILITY ZONES
+default   No            2           m5.xlarge                        us-east-1a
+mp1       No            2           m5.xlarge                        us-east-1a
+----
+
+. Review the output from the `rosa list machinepools` command to find the ID of the machine pool you want to scale and see the current number of replicas.
+
+. To change the scale, enter the following command and increase or decrease the number of replicas:
++
+[source,terminal]
+----
+$ rosa edit machinepool --cluster=<cluster_name> --replicas=<number_worker_nodes> <machinepool_ID>
+----
++
+.Example input
```

Comment:
Suggest changing to something similar with what you had in the previous file:

> This example changes the `default` machine pool ID scale to 3 replicas on a cluster named `mycluster`:

(Not sure if my description ^^ is entirely correct.)

Reply:
I removed this example because, upon reflection, it doesn't add value in this instance.